controllers/delete.controllers.js

```diff
@@ -0,0 +1,16 @@
+const { removeComment, checkCommentExists  } = require("../models/delete.models");
+
+exports.deleteComment = (req, res, next) => {
+    const { comment_id } = req.params
+    checkCommentExists(comment_id)
```

Comment:
I don't think it's too necessary for the checkCommentExists to actually return the value `true` and for you to check it here.
The very fact we've reached the `.then` block confirms the comment exists and an error hasn't been thrown.
To check if I'm right, you could try removing the if block below and see if it has any effect on your tests passing